hash/risc0/bench/host/benches/bench_main.rs

```diff
@@ -0,0 +1,44 @@
+use criterion::Criterion;
+// use clap::Parser;
+// use std::env;
+mod benchmarks;
+
+fn main() {
+    let mut criterion: criterion::Criterion<_>  = (Criterion::default().sample_size(10)).configure_from_args();
```

Comment:
Criterion benchmarks are useful for developing the code and running it standalone, but it's _not useful for us_. 

The main reason for that is that we want to also benchmark, and compare, many other things which are not implemented in Rust. So first of all, we cannot use Criterion for those, and second, criterion runs the code not once but several times. Clearly if we measure the overall runtime, that's not comparable with other kind of implementation running say once. 

Instead, we will provide our own benchmarking layer, and that's why there is a specification on exactly how we expect the individual benchmarks to behave - they need to be compatible with each other and with our system.

So: implement a separate CLI executable, which takes an argument (how many bytes or kilobytes of data), and runs the prover _exactly once_. That's should be built by "build.sh" and executed by "run.sh"